### DISCUSSION TITLE
Single line log file

### DISCUSSION THREAD

**Discussion by igor-teresco** [vectordotdev/vector#6259] [Q&A] [answered]
How to handle in vector log files written as one line? As now it stops on max_line_bytes and ignores the rest of the file...

**Comment by jszwedko** [comment, marked as answer]
Hi @igor-teresco . Could you provide some more context about what type of logs are being written to this file? Is it just one large JSON array as one line? Or something else?

**Reply by igor-teresco** [reply]
Hi, gladly! Log has XML structure like following:
`<?xml version="1.0" encoding="utf-8"?><log><diag contextId="08737cea-ea63-412a-9576-97af909772dc" level="42" timestamp="2020-12-18T13:56:32.083074Z">(Background Thread) StartupActivity not authenticated</diag><diag contextId="08737cea-ea63-412a-9576-97af909772dc" level="42" timestamp="2020-12-18T13:56:32.840987Z">(UI Thread) Set current activity as App.Authorization.Droid.SignInActivity</diag>`
New entries are added as tags: `<diag>text</diag>` and has no line feeds or other separators.

**Reply by jszwedko** [reply]
Aha, gotcha, thanks! So in this case, you'd like to read the `<diag>`s as they come in? Or just ingest the file once it is completely written with the closing tags?

**Reply by igor-teresco** [reply]
Ideally, I would like to read any tag as it will be closed, `<diag>` is most used, but other could show up.

**Reply by igor-teresco** [reply]
also one nuance - `<?xml version="1.0" encoding="utf-8"?><log>` is typical log start, but `</log>` could not be written if app is closed in bad way

**Reply by jszwedko** [reply]
Thanks for clarifying! I don't think vector has a way to currently support this so I opened an issue here: https://github.com/timberio/vector/issues/6322 . If you have additional details, I'd welcome them on that issue.
